ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -63,7 +68,11 @@ internal class PaywallViewModelImpl(
 
     override val state: StateFlow<PaywallViewState>
         get() = _state.asStateFlow()
+    override val actionInProgress: State<Boolean>
+        get() = derivedStateOf { _actionInProgress.value }
```

Comment:
Hmm I think `derivedStateOf` here is not needed? I think this could be:
```suggestion
        get() = _actionInProgress
```

Reply:
Oh good call 👍🏻